frontend/src/components/common/button/button.tsx

```diff
@@ -1,12 +1,15 @@
 import { FC } from 'common/types/types';
+import styles from './button.module.scss';
```

Comment:
We name styles folder as styles.module.scss
```suggestion
import styles from './styles.module.scss';
```